lib/qesapdeployment.pm

```diff
@@ -368,8 +369,9 @@ sub qesap_execute {
 
     push(@log_files, $exec_log);
     record_info('QESAP exec', "Executing: \n$qesap_cmd");
-    assert_script_run($qesap_cmd, timeout => $args{timeout});
+    my $exec_rc = script_run($qesap_cmd, timeout => $args{timeout});
     qesap_upload_logs();
+    return ($exec_rc);
```

Comment:
Why parenthesis ?

Reply:
API change to be managed in caller. Here some of them:
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/qesapdeployment/deploy.pm
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/qesapdeployment/destroy.pm
- https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/a58baa00222ae7c99ded4a82589d0ba4cdf07496/lib/trento.pm#L288


I'm fine about it

Reply:
Sorry... it was supposed to be without space return($exec_rc). I removed the parentheses completely.